### PR TITLE
Parser fixes and initial docs commit

### PR DESCRIPTION
**Pull request by jdent02**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+.. blenderseed Manual documentation master file, created by
+   sphinx-quickstart on Wed Mar 21 22:49:19 2018.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to the blenderseed Manual!
+==============================================
+
+	blenderseed is a plugin that allows Blender (version 2.79 or higher) to use the `appleseed <https://appleseedhq.net/>` rendering engine to produce imagery.  appleseed is a modern, fully programmable path tracer that integrates several cutting edge and open source features into a robust rendering package that is capable of rendering highly complex objects with dozens (or hundreds) of high resolution texures with deformation blur and depth of field.
```
Comment (dictoon) [inline docs/index.rst]: Move *open source* in the sentence:

> appleseed is a modern, open source, fully programmable path tracer that integrates several cutting edge features [...]

File: util.py [hunk 2]
```diff
@@ -151,11 +150,10 @@ def create_osl_dict(file, content=None):
                 currentElement['category'] = 'shader'
         else:
             if line.startswith("Default value"):
-                currentElement['default'] = line.split(" ")[-1].replace("\"", "")
+                currentElement['default'] = line.split(": ")[-1].replace("\"", "")
```
Comment (dictoon) [inline util.py]: Is the split by `": "` (with the additional space) intentional?

Comment (jdent02) [reply util.py]: Yes. it Removes the need for an additional strip() stage. The split with just ‘:’ would give you a leading space in front of the actual default value you need

File: docs/index.rst [hunk 3]
```diff
@@ -6,10 +6,8 @@
 Welcome to the blenderseed Manual!
 ==============================================
 
-	blenderseed is a plugin that allows Blender (version 2.79 or higher) to use the `appleseed <https://appleseedhq.net/>` rendering engine to produce imagery.  appleseed is a modern, fully programmable path tracer that integrates several cutting edge and open source features into a robust rendering package that is capable of rendering highly complex objects with dozens (or hundreds) of high resolution texures with deformation blur and depth of field.
+	blenderseed is a plugin that allows Blender (version 2.79 or higher) to use the `appleseed <https://appleseedhq.net/>`_ rendering engine to produce imagery.  appleseed is a modern, open source, fully programmable path tracer that integrates several cutting edge features into a robust rendering package that is capable of rendering highly complex objects with dozens (or hundreds) of high resolution texures with deformation blur and depth of field.
```
Comment (dictoon) [inline docs/index.rst]: There's a new `_` character after the link.

**Comment by dictoon** [comment]
Could you also combine your docs-related commits please? Thanks!

File: docs/index.rst (hunk 3 above)
Comment (jdent02) [reply docs/index.rst]: Yep that needs to be there for the link to work properly

Comment (dictoon) [reply docs/index.rst]: Alright!

**Pull request merged**